compiler/rustc_builtin_macros/src/format.rs

```diff
@@ -995,8 +995,9 @@ pub fn expand_preparsed_format_args(
             e.note(&note);
         }
         if let Some((label, span)) = err.secondary_label {
-            let sp = fmt_span.from_inner(span);
-            e.span_label(sp, label);
+            // FIXME(compiler-errors): Should we just suppress this secondary_label?
```

Comment:
I agree we should just suppress the secondary label. Its purpose is to point to the opening brace, which can't happen if we have to replace the span with `fmt_span`.